.env.example

```diff
@@ -18,3 +18,8 @@ export SIDEKIQ_PASSWORD=foo
 export MENDELEY_CONSUMER_KEY=foo
 export ERRBIT_API_KEY=foo
 export ERRBIT_HOST=localhost
+
+export SURVEY_EMAIL_USER=foobar
+export SURVEY_EMAIL_PASSWORD=foobar
+export SURVEY_EMAIL_ADDRESS=foobar
+export SURVEY_EMAIL_PORT=foobar
```

Comment:
Ah, I thought it was only about the sender address. Are you sure we still have that spam issue, even though our score looks pretty good? If so, I guess we need to go through GMail, then.